upstream-opm-builder.Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN GRPC_HEALTH_PROBE_VERSION=v0.2.1 && \
     wget -qO/bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-$(go env GOARCH) && \
     chmod +x /bin/grpc_health_probe
 
-FROM scratch
+FROM alpine
+RUN apk update && apk add bash ca-certificates
```

Comment:
Non-blocking: is `bash` necessary? Can the native `sh` be used directly?

Reply:
Probably. I'll take a look.

Reply:
Yeah bash is unnecessary. Removed it in the latest commit.

@jmrodri We can run `/bin/sh` in the entrypoint instead of `/bin/bash`